wavefront/server/apps/call_processing/call_processing/services/stt_service.py

```
@@ -214,6 +215,41 @@ def _create_elevenlabs_stt(api_key: str, parameters: Dict[str, Any]):
+
+        if 'sample_rate' in parameters and parameters['sample_rate']:
+            kwargs['sample_rate'] = parameters['sample_rate']
+        if 'endpoint_id' in parameters and parameters['endpoint_id']:
+            kwargs['endpoint_id'] = parameters['endpoint_id']
+        if (
+            'ttfs_p99_latency' in parameters
+            and parameters['ttfs_p99_latency'] is not None
+        ):
+            kwargs['ttfs_p99_latency'] = parameters['ttfs_p99_latency']
+
+        if 'language' in parameters and parameters['language']:
```

Ruff (0.15.6)

[warning] Unnecessary key check before dictionary access

Replace with `dict.get`

(RUF019)